Tic_tac_toe.py

```diff
@@ -0,0 +1,159 @@
+def create_board(number_of_rows, number_of_columns):
+    board = []
+    for i in range(0, number_of_rows):
+        board.append(["#"] * number_of_columns)
+    return board
+
+
+def print_board(board):
+    for row in board:
+        print("  ".join(row))
+
+
+def create_players():
+    return ['O', 'X']
+
+
+def get_validated_coordinates(player, board, number_of_rows, number_of_columns):
+    print("Now plays: " + player)
+    input_row = input("Please insert row number: ")
+    input_column = input("Please insert column number: ")
+    if input_row == "" or input_column == "":
+        print("Please insert valid coordinates!")
+        return get_validated_coordinates(player, board, number_of_rows, number_of_columns)
+    move_row = (int(input_row) - 1)
+    move_column = (int(input_column) - 1)
+    coordinates = [move_row, move_column]
+    if move_row > (number_of_rows - 1) or move_column > (number_of_columns - 1):
+        print("Your move is out of board!")
+        return get_validated_coordinates(player, board, number_of_rows, number_of_columns)
+    elif board[move_row][move_column] == "X" or board[move_row][move_column] == "O":
+        print("The field is already taken!")
+        return get_validated_coordinates(player, board, number_of_rows, number_of_columns)
+    return coordinates
+
+
+def mark_board_with_player_move(coordinates, player, board):
+    board[coordinates[0]][coordinates[1]] = player
+
+
+def is_current_player_winner(array, number_of_winning_marks, current_player):
```

Comment:
shouldn't you return `False` somewhere in this method?

Reply:
Done.